playground/mocks/idp/idx/challenge-poll.js

```diff
@@ -0,0 +1,5 @@
+const templateHelper = require('../../../config/templateHelper');
+
+module.exports = templateHelper({
+  path: '/idp/idx/challenge/poll',
```

Comment:
👍

Reply:
nit: should we call this `apiUrl` instead of `path`?

Reply:
use `path` to keep consistent with `path` from mock config.